Makefile

```diff
@@ -100,4 +86,4 @@ kubectl-config:
 	# Makefiles run recipes in sub-processes. A sub-process cannot modify the parent process's environment.
 	# The best I (@EngHabu) can think of at the moment is to output this for the user to eval in the
 	# parent process.
-	echo "export KUBECONFIG=$(KUBECONFIG):~/.kube/config:$(KUBE_CONFIG)/k3s/k3s.yaml"
+	echo "export KUBECONFIG=$(KUBECONFIG):~/.kube/config:$(FLYTE_DIR)/k3s/k3s.yaml"
```

Comment:
Does `flytectl sandbox start` add the kube context automatically? if so, we do not need this... right?
To maintain backward compat in case somebody is already using `make kubectl-config` in a script or something, we can make this just say:
```suggestion
	echo "deprecated: the setup/teardown targets automatically add/remove kubeconfig contexts"
```

Reply:
yes we don't need that